GeneticAlgorithmFW/src/main/java/net/hft/algodat/framework/geneticalgorithm/base/GeneticAlgorithm.java

```diff
@@ -134,9 +134,12 @@ public void run() {
         // GA- mainpart
         do {
             List<Individual> populationAfterSelection = this.selectionMethod.executeSelection(population);
-            this.crossoverMethod.executeCrossover(populationAfterSelection);
-            this.mutationMethod.executeMutation(populationAfterSelection);
-            this.replacementMethod.executeReplacement(population);
+            if (this.crossoverMethod != null)
```

Comment:
Please remove this lines of code. You destroyed the productive / test - changes. Validation is done by reflection